app/js/components/manageApps/ManageApps.jsx

```diff
@@ -159,25 +159,10 @@ export default class ManageApps extends React.Component {
       const apiBaseUrl = `/${applicationDistribution}/${url}/ws/rest`;
       this.requestUrl = '/v1/module/?v=full';
       axios.get(`${urlPrefix}/${apiBaseUrl}${this.requestUrl}`).then(response => {
-        response.data.results.forEach(data => {
-          installedModules.push({
-            appDetails: data,
-            appType: "module",
-            install: false
-          });
-        });
-        installedAddons = installedOwas.concat(installedModules);
```

Comment:
The code above that has been removed does not create a new set of data, it just modifies the data set. Those modifications are later used when checking which modules are installed to show if a install button is to be shown or a view button